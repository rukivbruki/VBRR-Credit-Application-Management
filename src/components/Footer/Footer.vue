<template>
  <div class="el-footer-text">
    <el-footer
      ><p>
        Powered by Â©
        <el-link href="https://www.egartech.ru/">EGAR Technology</el-link>
      </p>
    </el-footer>
  </div>
</template>

<script>
export default {
  name: 'Footer',
};
</script>

<style scoped>
.el-footer {
  text-align: center;
  line-height: 60px;
  font-family: Arial, serif;
  height: 2px;
  margin-top: 40px;
}
.el-footer > p {
  font-size: 12px;
  margin-top: 0;
}
.el-footer-text {
  color: white;
}
.el-link {
  font-size: 12px;
  text-decoration: underline;
}
</style>
