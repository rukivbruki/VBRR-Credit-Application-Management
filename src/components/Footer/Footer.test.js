import { shallowMount } from '@vue/test-utils';
import Footer from './Footer.vue';
describe('Footer.vue render', () => {
  test('renders correctly', () => {
    expect(shallowMount(Footer).element).toMatchSnapshot();
  });
});
