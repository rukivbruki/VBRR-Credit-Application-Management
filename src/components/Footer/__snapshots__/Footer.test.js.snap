// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer.vue render renders correctly 1`] = `
<div
  class="el-footer-text"
>
  <el-footer>
    <p>
       Powered by Â© 
      <el-link
        href="https://www.egartech.ru/"
      >
        EGAR Technology
      </el-link>
    </p>
  </el-footer>
</div>
`;
